src/components/SectionMessage/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import {
+    LabelPairedCircleCheckMdBoldIcon,
+    LabelPairedCircleExclamationMdBoldIcon,
+    LabelPairedCircleInfoMdBoldIcon,
+    LabelPairedTriangleExclamationMdBoldIcon,
+} from "@deriv/quill-icons";
+import clsx from "clsx";
+import { Text } from "../Text";
+import "./SectionMessage.scss";
+
+type TVariant = "warning" | "info" | "success" | "error" | "general";
+
+type SectionMessageProps = {
+    children: JSX.Element | string;
+    className?: string;
+    ctaSection?: JSX.Element;
+    title?: JSX.Element | string;
+    variant?: TVariant;
+};
+
+const VariantIcons: Record<Exclude<TVariant, "general">, JSX.Element> = {
```

Comment:
```suggestion
const VariantIcons: Record<Exclude<TVariant, "general">, IconTypes> = {
```

There's a ready made type for the icons from `deriv-com/quill-icons`

`import { IconTypes } from '@deriv/quill-icons';`

Reply:
sorted